app/src/componentLibrary/modal/modalLayout/modalLayout.jsx

```diff
@@ -125,4 +130,5 @@ ModalLayout.defaultProps = {
   modalSize: 'default',
   onClose: () => {},
   overlay: '',
+  isFieldsChanged: false,
```

Comment:
Seems like it should be `true` by default to allow closing modal windows when there is no any fields and reduce necessity of providing additional props.